src/Controller/Back/SignalementController.php

```diff
@@ -257,4 +258,44 @@ public function newDeleteSignalement(
 
         return $this->json($response, $response['status']);
     }
+
+    #[Route('/{uuid}/save-tags', name: 'back_signalement_save_tags', methods: 'POST')]
+    public function saveSignalementTags(
+        Signalement $signalement,
+        Request $request,
+        TagRepository $tagRepository,
+        EntityManagerInterface $entityManager
+    ): Response {
+        $this->denyAccessUnlessGranted('SIGN_EDIT', $signalement);
+
+        if (
+            $this->isCsrfTokenValid('signalement_save_tags', $request->request->get('_token'))
+        ) {
+            $tagIds = $request->request->get('tag-ids');
+            $tagList = explode(',', $tagIds);
+            foreach ($signalement->getTags() as $existingTag) {
+                if (!\in_array($existingTag->getId(), $tagList)) {
+                    $signalement->removeTag($existingTag);
+                }
+            }
+            if (!empty($tagIds)) {
+                foreach ($tagList as $tagId) {
+                    $tag = $tagRepository->find($tagId);
```

Comment:
faire un find by id + territoire par sécurité, + vérifier en dessous qu'on a bien un tag not null pour passer au traitement suivant (j'ai eu un crash pendant mes tests sur ça)

Reply:
OK !